src/IIIF.php

```diff
@@ -9,24 +9,58 @@ class IIIF {
 
     private $pid;
     private $xpath;
+    private $object;
     private $type;
     private $url;
 
-    public function __construct($pid, $mods, $object)
+    public function __construct($pid, $mods, $object, $model = null)
     {
 
-        $model = simplexml_load_string($object['body'])->objModels->model;
+        if (!$model) {
+            $model = simplexml_load_string($object['body'])->objModels->model;
+        }
 
         $this->pid = $pid;
         $this->mods = $mods;
+        $this->object = $object;
         $this->xpath = new XPath($mods);
         $this->type = self::determineTypeByModel($model);
 
         $this->url = Utility::getBaseUrl();
 
     }
 
-    public function buildPresentation ()
+    public function buildCollection ()
+    {
+        $id = $this->url . str_replace('?update=1', '', $_SERVER["REQUEST_URI"]);
+
+        $collection['@context'] = ['https://iiif.io/api/presentation/3/context.json'];
```

Comment:
This shouldn't be an array.  It should be key, value.

Reply:
> @mathewjordan Sorry for the delay. This seems pretty good. I have one minor change and a few questions.
> 
>     1. Is there an intent right now for the colleciton manifest to reference collections that belong to it or it belongs to?  If so, that doesn't seem to work yet.
> 
>     2. Is there an expectation that this just follows the MUSTs or is the expectation that it also follows the SHOULDs? If it's the latter, we're missing these things: metadata, summary, provider, thumbnail.

@markpbaggett 

1. For now, I just want a single level collection, with all items having type Manifest. At some point, perhaps in the fall, we can start looking at how we would want to handle collections embedded within collections.
2. Yes, this is just MUSTs for now. 

Per https://iiif.io/api/presentation/3.0/#46-linked-data-context-and-extensions arrays are allowed as long as the last item in that array is `http://iiif.io/api/presentation/3/context.json` . Although not necessary for our implementation I do this because the presentation validator screams at me otherwise.

> The value of the @context property must be either the URI http://iiif.io/api/presentation/3/context.json or a JSON array with the URI http://iiif.io/api/presentation/3/context.json as the last item. Further contexts, such as those for local or registered extensions, must be added at the beginning of the array.

Reply:
@mathewjordan good to know and an interesting choice for the spec.  as a fan of static typing, i wish they would have just gone with the value always being an array but i'll get over it.